README.md

```diff
@@ -1,68 +1,119 @@
 
-## Tutorial
+The code is based on https://github.com/dternyak/React-Redux-Flask and https://github.com/creativetimofficial/material-dashboard-react.
 
-## 1. Setting Up The Project
+## How to Run
+
+## 1. Setting up Azure Services
+First, sign up for a free [Azure](https://azure.microsoft.com/en-us/free/) account if you don't already have one. Sign into https://portal.azure.com.
+
+[Create a resource group](https://github.com/lilyjma/azurethings/blob/master/createResourceGroup.md) to store the resources that you'll be using here--Azure Cosmos DB and Key vault. Then follow the instructions in the links below to create each resource:
+
+(Remember to store them in the resource group you created; this will make it easier to clean up the resources in the future.)
+
+1. [Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/create-cosmosdb-resources-portal#create-an-azure-cosmos-db-account)
+   1. When creating the database, name it 'team_standup'. For this app, you also need two containers named 'tasks' and 'users'. The [partition key](https://docs.microsoft.com/en-us/azure/cosmos-db/partitioning-overview#choose-partitionkey) for both is '/id'. 
+2. [Key Vault](https://docs.microsoft.com/en-us/azure/key-vault/quick-create-portal#create-a-vault)
+   1. This will store the credentials for the resources this app uses. For example, it'll store the key to the cosmos database. This way, you don't reveal any key in your code. 
+   2. You'll add two secrets called 'cosmosKey' and 'cosmosURI' to Key Vault to hold the cosmos key and URI respectively. To find these, click into the Cosmos DB account created, go to 'Keys' tab and get the Primary Key and the URI. 
+
+## 2. Getting Access to Key Vault
+To make a long story short, you need a service principal to have access to key vault. The service principal serves as an application ID that is used during the authorization setup for access to other Azure resources. We'll use a Web App instance as our service principal. To do that, we create an App Service Plan, then a Web App instance, then make that our service principal. We can do all of this on Cloud Shell (click >_ on the top right hand corner in Portal to open). 
+
+1. Create App Service Plan : 
+   
+    ```az appservice plan create --name myServicePlanName --resource-group myResourceGroup --location westus```
+
+    There are locations other than westus. A json object will pop up when the command is done. 
+
+2. Create a Web App instance : (Note that the app name must be unique.)
+
+    ```az webapp create --name myUniqueAppName --plan myServicePlanName --resource-group myResourceGroup```
+
+3. Make the web app a service principal : 
+    
+    ```az ad sp create-for-rbac --name http://my-applications-url --skip-assignment```
+
+    Use your web app's url. To find that, click on the app in Portal, go to the 'Overview' tab and look for 'URL' on the top right portion of the page. It looks something like https://myUniqueAppName.azurewebsites.net. 
```

Comment:
What do you mean use your web apps url?  In the above or below?  Please be explicit in everything you ask the user to do.